cli/src/test/java/com/devonfw/tools/ide/log/IdeTestLoggerAssertion.java

```diff
@@ -23,69 +23,70 @@ public IdeTestLoggerAssertion(List<IdeLogEntry> entries, IdeLogLevel level) {
   /**
    * @param message the expected {@link com.devonfw.tools.ide.log.IdeSubLogger#log(String) log message}.
    */
-  public void hasMessage(String message) {
+  public IdeTestLoggerAssertion hasMessage(String message) {
 
     fulfillsPredicate(e -> e.message().equals(message), PredicateMode.MATCH_ONE, "Could not find log message equal to '" + message + "'");
+    return this;
```

Comment:
Why are we returning `this` only here instead of the result of `fullfillsPredicate`?

Reply:
Good question. Aligned the code...